src/libraries/System.Drawing.Common/src/Interop/Windows/Interop.Gdi32.cs

```diff
@@ -179,14 +179,16 @@ internal unsafe struct BITMAPINFO_FLAT
         [NativeMarshalling(typeof(Marshaller))]
 #endif
         [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
-        internal sealed class DOCINFO
+        internal struct DOCINFO
         {
             internal int cbSize = 20;
             internal string? lpszDocName;
             internal string? lpszOutput;
             internal string? lpszDatatype;
             internal int fwType;
 
+            public DOCINFO() { }
```

Comment:
Why add the constructor? Isn't this implicit?

Reply:
Because it is now a struct with a field initializer, I had to [add an explicitly declared constructor](https://learn.microsoft.com/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%207#6).

Reply:
Ah yes, thanks. :)